### PR TITLE
Fix comments title and comments view

### LINKED ISSUE #142: Format comments details as task details
## Expected Behavior
The details should look like: (no need for the description title)
image

## Actual Behavior
image

### PR DESCRIPTION
**Pull request by AvivLiberman**
Fixes #142 

This PR is for formatting the comments as tasks and remove comments title if task has 0 comments.
Very thin PR only some HTML changes.

## Changes you made
- Added "if" check for length of given comments on the template and removed title accordingly.
- Formatted comments the same as tasks

### CODE REVIEW

**Review by orzionpour — DISMISSED** [review]

**Review by Asafh7 — APPROVED** [review]

File: tasks/templates/tasks/view_single_task.html [hunk 1]
```diff
@@ -1,47 +1,53 @@
...
-                        <small class="text-muted">
-                            {{ comment.created_date|date:"F d, Y" }}
-                        </small>
-                    </div>
-                    <p class="article-content">{{ comment.description|truncatechars:400 }}</p>
-                </div>
-            </article>
-        {% endfor %}
-    {% endif %}
+{% if user.user.is_authenticated%}
```
Comment (kobihk) [inline tasks/templates/tasks/view_single_task.html:3]: please update the indentation 
with correct indentation, the code will be more readable

File: tasks/templates/tasks/view_single_task.html [hunk 2]
```diff
@@ -1,47 +1,53 @@
...
+      <small class="text-muted">
+        <b>Created on: </b>{{ comment.created_date|date:"F d, Y" }} |
+        <b>Created by: </b>{{ comment.appUser.user.first_name|truncatechars:20}} {{ comment.appUser.user.last_name|truncatechars:20 }}
+      </small>
+    </div>
+  </div>
+</article>
+{% endfor %}
+{% endif %}
+{% endif %}
```
Comment (kobihk) [inline tasks/templates/tasks/view_single_task.html:52]: same ^^

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by avraham1992 — APPROVED** [review]

File: tasks/templates/tasks/new_task.html [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
-        </div>
-    {% elif user.user.is_authenticated %}
-        This page is restricted for managers only.
-    {% else %}
-        This page is restricted. Are you an existing user? <a href="{% url 'login' %}"> Login</a>.
-    {% endif %}
-
-{% endblock content %}
-
+{% extends 'base.html' %} {% block content %} {% if user.is_manager %}
```
Comment (kobihk) [inline tasks/templates/tasks/new_task.html:1]: if you have a block of code that you have open "command" like: 
- "{% if user.is_manager %}" or "{% block content %}"

I expect to see the close "command" of it in the same indentation as you did in:
tasks/templates/tasks/view_single_task.html

```suggestion
{% extends 'base.html' %} 
{% block content %} 
    {% if user.is_manager %}
```

File: tasks/templates/tasks/new_task.html [hunk 4]
```diff
@@ -1,31 +1,30 @@
...
+          type="submit"
+          value="Create Task"
+        />
+      </form>
+    </div>
+  </div>
+</div>
+{% elif user.user.is_authenticated %} This page is restricted for managers only.
+{% else %} This page is restricted. Are you an existing user?
+<a href="{% url 'login' %}"> Login</a>. {% endif %} {% endblock content %}
```
Comment (kobihk) [inline tasks/templates/tasks/new_task.html:30]: same ^^
```suggestion
<a href="{% url 'login' %}"> Login</a>. 
    {% endif %} 
{% endblock content %}
```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by aradz1997 — COMMENTED** [review]
can you explain what did you change in 'new_task.html' ?

**Comment by AvivLiberman** [comment]
> can you explain what did you change in 'new_task.html' ?

You are right probably some as a result of the merge, will change it in the evening

File: tasks/templates/tasks/new_task.html (hunk 3 above)
Comment (AvivLiberman) [reply tasks/templates/tasks/new_task.html]: changed by mistake please ignore

File: tasks/templates/tasks/new_task.html (hunk 4 above)
Comment (AvivLiberman) [reply tasks/templates/tasks/new_task.html]: changed by mistake please ignore

**Review by kobihk — APPROVED** [review]

**Review by orzionpour — APPROVED** [review]
lgtm

**Review by aradz1997 — APPROVED** [review]

**Pull request merged**